openhcl/virt_mshv_vtl/src/processor/snp/mod.rs

```diff
@@ -1370,6 +1371,11 @@ impl UhProcessor<'_, SnpBacked> {
             }
 
             SevExitCode::NPF if has_intercept => {
+                // TODO SNP: This code needs to be fixed to not rely on the
```

Comment:
Are you sure? I thought Alex said this was necessary. Doesn't the legacy HCL do this, too?

Reply:
I talked with Jon offline and he said we shouldn't be relying on the hypervisor for anything - the hardware exit info on both SNP and TDX should have what we need. I haven't looked at the legacy HCL in detail but he said that's what he believes to be the case after some refactoring.